internal/thelma/clients/sherlock/state_writer.go

```diff
@@ -86,6 +86,15 @@ func (c *Client) CreateEnvironmentFromTemplate(templateName string, options terr
 			Enabled: utils.Nullable(options.AutoDelete.Enabled),
 		}
 	}
+	if options.StopSchedule.Enabled {
```

Comment:
Do we need some kind of guard against a bee with only a start or stop schedule but not both?

Reply:
No, that case is specifically allowed by all three systems. I was thinking mainly that only a stop schedule, so starts would only be on demand, might be reasonable for some folks (me).